C/ColPack/build_tarballs.jl

```diff
@@ -18,14 +18,17 @@ autoreconf -vif
 
 mkdir build
 cd build
-CC=gcc
-CXX=g++
 ../configure --enable-examples --build=${MACHTYPE} --host=${target}
 make -j${nproc}
 
 mkdir -p ${bindir}
 cp ColPack${exeext} ${bindir}/ColPack${exeext}
-g++ -shared $(flagon -Wl,--whole-archive) libcolpack.a $(flagon -Wl,--no-whole-archive) -lgomp -o ${libdir}/libcolpack.${dlext}
+
+if [[ "${target}" == *apple* ]] || [[ "${target}" == *freebsd* ]]; then
+  ${CXX} -shared $(flagon -Wl,--whole-archive) libcolpack.a $(flagon -Wl,--no-whole-archive) -lomp -o ${libdir}/libcolpack.${dlext}
+else
+  ${CXX} -shared $(flagon -Wl,--whole-archive) libcolpack.a $(flagon -Wl,--no-whole-archive) -lgomp -o ${libdir}/libcolpack.${dlext}
+fi
```

Comment:
Minimise duplication:
```suggestion
if [[ "${target}" == *apple* ]] || [[ "${target}" == *freebsd* ]]; then
    LDFLAGS=-lomp
else
    LDFLAGS=-lgomp
fi
${CXX} -shared $(flagon -Wl,--whole-archive) libcolpack.a $(flagon -Wl,--no-whole-archive) ${LDFLAGS} -o ${libdir}/libcolpack.${dlext}
```